.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: python
+arch:
+  - amd64
+  - ppc64le
 python:
   - "2.7"
   - "3.4"
   - "3.5"
   - "3.6"
   - "3.7"
+before_install:
+  - sudo chown -Rvf $USER:$GROUP ~/.cache/pip/wheels
```

Comment:
Is this necessary? I never noticed a permissions problem in the build before. Have you observed a specific error related to this?